deploy/example/backup-app/README.md

```diff
@@ -0,0 +1,56 @@
+# Example Backup Application
+
+This document provides steps to install a backup application that communicates with the snapshot-metadata service to utilize the Changed Block Tracking (CBT) feature. You can use this example as a reference when deploying the backup application with the required permissions to use the CBT feature.
```

Comment:
I suggest changing the slant of this document to `Authorizing a Backup Application`

The directory should consist of a sample ServiceAccount and ClusterRoleBinding only.